content/1_General/Fast_IO.mdx

```diff
@@ -64,7 +64,8 @@ less than $10^9+7$.
 Randomly generating test data results in input and output files each ~10MB
 large. It is possible to see input files this large (the 11th input file for
 [Robotic Cow Herd](http://www.usaco.org/index.php?page=viewproblem2&cpid=674) is
-~10.3MB large), though not output files (the largest I recall is due to
+~10.3MB large), though not output files (an extremely larger one is one for a
```

Comment:
@SansPapyrus683 the current wording doesn't make sense

Reply:
large instead of larger?

Reply:
I prefer my original wording

Reply:
first person seems kinda informal?